src/components/Title/index.js

```diff
@@ -12,7 +12,9 @@ const Title = ({title, subTitle, smaller, isFullScreen, className}) =>
       },
       className
     )}>
-    <div className={style.title}>{title}</div>
+    <div className={style.title}>
+      <span className={style.titleSpan}>{title}</span>
```

Comment:
So this is the only example so far of `em` making things awkward.

Since we change the font-size of `title`, working out the `margin-bottom` it does in terms of `em` is a bit confusing :/

So here I've just split it out into a separate `span`, so our `margin-bottom` can be independent of the `font-size` change.

It's pretty ugly :/ But maybe not so bad, if examples of this being a problem are as few and far between as this little foray into fixing up the units has been

Reply:
Yeah. So as I say, hopefully it'll be something that comes up infrequently (it looks like it, from this pr, at least)
But when it does come up, either font-size on leaf nodes, or just not using `@unit`, or using it but accepting that it will be some ratio different in that context

If we go for `@unit` being `rem` then we don't have this constraint, but it means that clients cannot change the size without us using CSS vars or something